COPYING

```diff
@@ -1,6 +1,19 @@
 This repository contains the generator for the MAVLink protocol. The generator itself is
-(L)GPL v3 licensed, while the generated code is subject to different licenses:
-
+(L)GPL v3 licensed, while the generated code is subject to the MIT license.
```

Comment:
I don't understand the need for this change, what is written below is much more clear - especially since what it says below is that the generated code is only under MIT license if the product where it is included is licensed as MIT itself.

Reply:
The legal department of Volkswagen requires that the copyright information be available in a single file on the same repo as the source code. This was my attempt at achieving that, having it all in the same file as the license.

The MIT license requires to have copyright information in the license, we do not have that now.

Reply:
> The legal department of Volkswagen requires that the copyright information be available in a single file on the same repo as the source code.

A single file doesn't mean in the same file as the license.

> The MIT license requires to have copyright information in the license, we do not have that now.

No, it doesn't. What it says is that the Copyright notice needs to be included in any copy of the code. If there is no Copyright notice then there isn't anything to copy and anyone would still be complying with the license.